src/components/DatePickerDialog/index.tsx

```diff
@@ -0,0 +1,208 @@
+import React, {
+  ChangeEventHandler,
+  FC,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import { format, isValid, parse } from 'date-fns';
+import FocusTrap from 'focus-trap-react';
+import { DayPicker } from 'react-day-picker';
+import { usePopper } from 'react-popper';
+import styled from 'styled-components';
+import CalendarToday from '@material-ui/icons/CalendarToday';
```

Comment:
My understanding was that we wanted to get rid of the @material-ui dependency. Introducing a new icon here is counterproductive in this sense. Could we maybe download the icon and import it as an asset?

Reply:
Good point, I had forgotten that move away from the Material Icons dependency. It's changed now!